### DISCUSSION TITLE
Specification website https://age-encryption.org/ security error

### DISCUSSION THREAD

**Discussion by sj4nes** [FiloSottile/age#262] [Q&A]
Firefox, Safari and Brave browsers all complain that there is something wrong with https://age-encryption.org/ 's certificate and refuse to open the site.

**Comment by FiloSottile** [comment]
Regrettably, some "security" software vendors think that blocking the specification website of an encryption format will do something against ransomware that uses that encryption format. It's as incompetent as it sounds.

Moreover, these vendors engage in HTTPS MitM when trying to block websites, which age-encryption.org is configured to prevent through HSTS. This is a terrible practice that only trains users to skip certificate errors, noticeably reducing their security.

Honestly, I recommend reconsidering whatever security product you are using. You can probably get a hint from the fake certificate or from `curl http://age-encryption.org`. If you feel like it, you could also file a ticket for an incorrect classification.

**Comment by Fastidious** [comment]
> Firefox, Safari and Brave browsers all complain that there is something wrong with https://age-encryption.org/ 's certificate and refuse to open the site.

For what is worth, I am using Safari (Version 14.0.3 (16610.4.3.1.7)) and I don't have any problems with certificates on the referred site.

**Reply by FiloSottile** [reply]
Yeah this is almost certainly related to some kind of firewall or AV on @sj4nes's network.

**Reply by sj4nes** [reply]
Yeesh. 

Yeah, I can't even spin up a Tor circuit so it is most likely this.  I also found it insta-killed a `curl` installation process for oh-my-zsh.

**Reply by sj4nes** [reply]
DNS filtering was the culprit.

**Reply by FiloSottile** [reply]
Glad you found the source! I want to point out that 1) DNS filtering should not produce fake HTTPS certificates, it's a bad practice and 2) there is no good reason to block age-encryption.org. You probably can't do anything about (1) except switch to a better vendor, but you can probably report (2) and get it fixed.
